### PR TITLE
Feat/find

### PR DESCRIPTION
**Pull request by NyaGarcia**

### CODE REVIEW

File: src/constants/main.constants.ts [hunk 1]
```diff
@@ -47,3 +47,6 @@ export const BANNER = `-----                    __   .__.__  .__
 --        |___|  /   __/|__|_ \\__|____/____/
 -------        \\/|__|        \\/
 `;
+
+export const STREAM_ENCODING = 'utf8';
+export const WINDOWS = 'win32';
```
Comment (zaldih) [inline src/constants/main.constants.ts]: Ambiguous Name. Maybe "WINDOWS_PLATFORM" will better?

Comment (NyaGarcia) [reply src/constants/main.constants.ts]: Change as you wish

**Comment by NyaGarcia** [comment]
Upgrade search algorithm to use two different strategies:
-Native find command on Linux OS
-Go script on Windows OS
Performance increases greatly

**Comment by zaldih** [comment]
YAY!!!!

**Pull request merged**